tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AppProvider>
+    <Page title="Judge.Me Reviews">
+      <Layout>
+        <LayoutSection>
+          <Button @click="toggleCollapsibleActive" style="margin-bottom: 20px;">{{ formButtonMessage }}</Button>
+          <Collapsible :open="collapsibleActive" :class="{ collapsibleActive: collapsibleActive }">
+            <Card>
+              <ReviewForm @add-review="addReview" :key="formReset" />
+            </Card>
+          </Collapsible>
+          <Checkbox label="Unverified reviews only" v-model="unverifiedOnly" />
+          <Divider />
+          <EmptyState
+            v-if="!reviews || reviews.length < 1"
+            heading="No reviews found">
+            <Text as="p">Add a review above or reload the page</Text>
+          </EmptyState>
+          <ReviewCard
+            v-for="(review, index) in reviews"
+            :key="review.id"
+            :reviewObject="review"
+            :unverifiedOnly="unverifiedOnly"
+            @remove-review="removeReview(review.id, index)"
+            @verify-review="verifyReview(review.id, index)"
+          />
+        </LayoutSection>
+      </Layout>
+    </Page>
+  </AppProvider>
+</template>
+
+<script setup>
+import { computed, onMounted, ref } from 'vue';
+import ReviewCard from './components/ReviewCard.vue';
+import ReviewForm from './components/ReviewForm.vue';
+
+const reviews = ref(null);
+const collapsibleActive = ref(false);
+const formButtonMessage = ref('Add new review');
+const unverifiedOnly = ref(false);
+const formReset = ref(0);
+
+const toggleCollapsibleActive = () => {
+  collapsibleActive.value = !collapsibleActive.value;
+  collapsibleActive.value ? formButtonMessage.value = 'Hide form' : formButtonMessage.value = 'Add a new review';
+};
+
+const fetchReviews = () => {
+  const url = new URL('https://663e3cbae1913c47679705a2.mockapi.io/api/v1/reviews');
+  url.searchParams.append('sortBy', 'createdAt');
+  url.searchParams.append('order', 'desc');
+
+  fetch(url)
+    .then(response => response.json())
+    .then(data => reviews.value = data)
+}
+
+const addReview = () => {
+  fetchReviews();
```

Comment:
Should we not wait for `fetchReviews` to be resolved before moving onto the next step?

Reply:
Here I don't think it's strictly necessary as the next steps aren't dependant on the response from that function (it's just closing the form), but I appreciate this could be done if required.